tests/samples/input/ocaml.ml

```diff
@@ -495,3 +495,16 @@ let add_as_fun_multiline = fun x ->
   x
 
 let add_as_fun_one_line = fun x -> x
+
+(* Showcase the usage of operator bindings *)
+let greetings =
+  let (let*) = Option.bind
+  and (and*) a_opt b_opt =
+    match (a_opt, b_opt) with
+    | (Some a, Some b) -> Some (a, b)
+    | _ -> None
+  in
+  let* msg1 = Some "hello "
```

Comment:
I would add an example where the

```ocaml
let* x =
  something or some other
in
```

is multiline like this.

Reply:
done in latest version